src/probability/integration.lean

```diff
@@ -118,4 +120,132 @@ lintegral_mul_eq_lintegral_mul_lintegral_of_independent_measurable_space
   (measurable_iff_comap_le.1 h_meas_f) (measurable_iff_comap_le.1 h_meas_g) h_indep_fun
   (measurable.of_comap_le le_rfl) (measurable.of_comap_le le_rfl)
 
+/-- This proves that if `f` and `g` are independent and almost everywhere measurable,
```

Comment:
Removing "This proves that"/"This shows that" from all the docstrings would make them less verbose while keeping the same information.

Reply:
I agree, I was just matching the style of the rest of the file but I can update the other doctrings as well.